OrbitCore/LinuxEventTracer.cpp

```diff
@@ -0,0 +1,262 @@
+//-----------------------------------
+// Copyright Pierric Gimmig 2013-2019
+//-----------------------------------
+
+#include "ContextSwitch.h"
+#include "OrbitProcess.h"
+#include "PrintVar.h"
+#include "LinuxPerfUtils.h"
+#include "LinuxPerfEvent.h"
+#include "LinuxEventTracer.h"
+#include "LinuxUtils.h"
+#include "Utils.h"
+#include "Capture.h"
+#include "TimerManager.h"
+
+#include <assert.h>
+#include <functional>
+#include <queue>
+#include <vector>
+
+
+//-----------------------------------------------------------------------------
+LinuxEventTracer::LinuxEventTracer()
+{
+    //m_Callback = a_Callback ? a_Callback : [this](const std::string& a_Buffer)
```

Comment:
I would either remove code that is commented out or add a comment why it is commented out and when it should be reenabled.